lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -279,95 +227,6 @@ def test_get_entrance_exam_content(self):
         assert exam_chapter is None
         assert user_has_passed_entrance_exam(self.request.user, self.course)
 
-    def test_entrance_exam_requirement_message(self):
```

Comment:
There only appear to be a couple of Entrance Exam tests in the MFE. Do we need to port some of these tests to the MFE?

Reply:
It would probably be nice to add more entrance exam tests to the MFE, but:
- I tried to only remove the UI tests here, leaving the business logic tests in place.
- This means that these removed tests were in theory not providing any value anyway, just testing UI that no one saw.
- We (2U) don't use entrance exams, so on the one hand it's not a high priority, but on the other, we won't notice issues when using it so tests are definitely nice.

So I see it as a different effort than this removal, since in theory I'm only removing tests that weren't giving us any value anyway. But more tests in the MFE ain't bad also.